webauthn/helpers/parse_registration_credential.py

```diff
@@ -0,0 +1,21 @@
+from typing import Union
+from pydantic import ValidationError
+
+from .exceptions import InvalidRegistrationResponse
+from .structs import PYDANTIC_V2, RegistrationCredential
+
+
+def parse_registration_credential(json_val: Union[str, bytes, bytearray]) -> RegistrationCredential:
```

Comment:
Similar suggestion as above, I think this method should be called `parse_registration_credential_json()`.

Reply:
Fixed as well.